IDE/MDK5-ARM/Projects/wolfSSL-Full/shell.c

```diff
@@ -580,15 +561,6 @@ static void command_invoke(void const *args)
 
     if(iteration > 1)
     for_iteration = 1 ;
-    osDelay(20000) ;
-    #ifdef HAVE_KEIL_RTX
-        wc_UnLockMutex((wolfSSL_Mutex *)&command_mutex) ;
```

Comment:
With HAVE_KEIL_RTX this removes the unlock but still has a lock command at the beginning of command_invoke. Which looks like it will leave it permanently locked.